Description: Don't add executables to cmake exports
 CMake exports are used by other packages that compile
 and link against dcmtk. Because Debian moves some of
 these executables and also doesn't install the test
 executables, this import may fail leading to failure
 to configure the according package. 
Bug-Debian: https://bugs.debian.org/803304
Author: Gert Wollny <gw.fossdev@gmail.com>
--- a/CMake/dcmtkMacros.cmake
+++ b/CMake/dcmtkMacros.cmake
@@ -60,11 +60,11 @@
         endif()
 
         # Collect executable as part of global DCMTK_EXECUTABLE_TARGETS property
-        set_property(GLOBAL APPEND PROPERTY DCMTK_EXECUTABLE_TARGETS ${PROGRAM})
+        #set_property(GLOBAL APPEND PROPERTY DCMTK_EXECUTABLE_TARGETS ${PROGRAM})
 
         # declare installation files, also export DCMTKTargets.cmake
         install(TARGETS ${PROGRAM}
-                EXPORT DCMTKTargets
+        #        EXPORT DCMTKTargets
                 COMPONENT bin
                 DESTINATION ${CMAKE_INSTALL_BINDIR})
     endif()
